src/webrtc/call.ts

```diff
@@ -774,6 +787,49 @@ export class MatrixCall extends EventEmitter {
         this.sendVoipEvent(EventType.CallReject, {});
     }
 
+    /**
+     * Adds a video track to a voice call
+     * @returns new mute state
+     */
+    private async upgradeCall(
+        audio: boolean, video: boolean,
+    ): Promise<void> {
+        // We don't do call downgrades
+        if (!audio && !video) return;
+        if (!this.opponentSupportsSDPStreamMetadata()) return;
+
+        try {
+            const upgradeAudio = audio && !this.hasLocalUserMediaAudioTrack;
+            const upgradeVideo = video && !this.hasLocalUserMediaVideoTrack;
+            logger.debug(`Upgrading call audio=${upgradeAudio} video=${upgradeVideo}`);
```

Comment:
The log line isn't clear that the upgrade variables are being used rather than the target audio/video state, so something like this would be appreciated: 
```suggestion
            logger.debug(`Upgrading call: audio?=${upgradeAudio} video?=${upgradeVideo}`);
```